captcha/fields.py

```diff
@@ -23,7 +26,7 @@ class ReCaptchaField(forms.CharField):
     }
 
     def __init__(self, public_key=None, private_key=None, use_ssl=None,
-                 attrs=None, *args, **kwargs):
+                 wizard=None, attrs=None, *args, **kwargs):
```

Comment:
I missed this before. If you introduce a new argument then any existing code that subclasses this class will break. You will have to do

    wizard = kwargs.get("wizard", None)

The nice thing about kwargs is you can change your method signatures and not break other code.